VimeoUpload/Upload/Networking/Old Upload/VimeoSessionManager+OldUpload.swift

```diff
@@ -117,7 +117,7 @@ extension VimeoSessionManager
     {
         let request = try (self.requestSerializer as! VimeoRequestSerializer).uploadVideoRequestWithSource(source, destination: destination)
         
-        let task = self.uploadTaskWithRequest(request, fromFile: source, progress: progress, completionHandler: { [weak self] (response, responseObject, error) -> Void in
+        let task = self.uploadTaskWithRequest(request, fromFile: source, progress: {(progress) -> Void in }, completionHandler: { [weak self] (response, responseObject, error) -> Void in
```

Comment:
Same here too, can you describe why this change is necessary? Seems like we should keep this logic as it was